docs/tutorials/extending_admin_ui/5_creating_a_udw_tab.md

```diff
@@ -435,12 +435,10 @@ Place the compiled files in `src/EzSystems/ExtendingTutorialBundle/Resources/pub
 
     If you are not familiar with compiling React modules, you can use the following steps:
 
-    1. Copy the `webpack.*.js` files from `vendor/ezsystems/ezplatform-page-builder/src/bundle/ui-dev`.
-    1. Copy the `package.json` and `.babelrc` files from the same folder to your project.
-    1. Change values in the `entry` property in `webpack.*.js` to `./src/images.panel.js`, also define module name `ImagesPanel` instead of `PageBuilder` or `Timeline`.
-    1. In the terminal, run `npm install`.
+    1. Copy the `webpack.*.js`, `package.json`, and `.babelrc` files from `vendor/ezsystems/ezplatform-admin-ui-modules` to `src/EzSystems/ExtendingTutorialBundle/Resources/ui-dev`.
+    1. Replace the existing modules in the `entry` property in `webpack.common.js` with `ImagesPanel: './src/images.panel.js',`.
```

Comment:
```suggestion
    1. Replace the existing modules in the `entry` property in `webpack.common.js` with `ImagesPanel: './src/images.panel.js'`.
```

Reply:
The comma is actually part of code, cf. https://github.com/ezsystems/ezplatform-admin-ui-modules/blob/master/webpack.common.js#L9

Reply:
In that case, sorry for the unnecessary comment. :)